vscode-extension/static/untitled.aiconfig.json

```diff
@@ -11,21 +12,21 @@
     },
     "default_model": "gpt-3.5-turbo"
   },
-  "description": "Fill out a description about your AIConfig here.",
+  "description": "Enter a description for this AIConfig. Learn about features in our [docs](https://aiconfig.lastmileai.dev/docs/category/overview)!",
```

Comment:
"Enter a description for this AIConfig. You can also check out [our docs](https://aiconfig.lastmileai.dev/docs/category/overview) to learn more!